waitgroup_test.go

```diff
@@ -98,5 +98,46 @@ func TestWaitGroup(t *testing.T) {
 			require.Panics(t, wg.Wait)
 			require.Equal(t, int64(2), i.Load())
 		})
+
+		t.Run("is caught by waitsafe", func(t *testing.T) {
+			t.Parallel()
+			var wg WaitGroup
+			wg.Go(func() {
+				panic("super bad thing")
+			})
+			p := wg.WaitSafe()
+			require.Contains(t, p.Error(), "super bad thing", p.Error())
```

Comment:
nit: we can compare the panic value directly rather than checking the stringified error message. 

Also, arguments 4+ for `require.Contains()` are to optionally add more information to the failure message. The error will already be included in the message if it fails, so there's no need for the optional args here. 

```suggestion
			require.Equal(t, p.Value, "super bad thing")
```

Reply:
Oops. Thank you! Fixed.